deployments/kubernetes/install/helm/istio-cni/values.yaml

```diff
@@ -13,6 +13,7 @@ logLevel: info
 # defaults:
 cniBinDir: /opt/cni/bin
 cniConfDir: /etc/cni/net.d
+kubernetesCNIConfDir: /etc/kubernetes/cni/net.d
```

Comment:
The PR is about openshift but looks like this impacts everyone. What is the impact of this change for GKE/kind/EKS/others?